diff --git a/dist/schemas.js b/dist/schemas.js
index 060d0b98b76515e4f0a1cd9edb3c3f5d31597540..9afd60daa86510b5ace13ad26232c644ff0a4f71 100644
--- a/dist/schemas.js
+++ b/dist/schemas.js
@@ -65,7 +65,6 @@ const LayoutPropsSchema = z.strictObject({
   banner: reactNode.optional().meta({
     description: "Rendered [`<Banner>` component](/docs/built-ins/banner). E.g. `<Banner {...bannerProps} />`"
   }),
-  children: reactNode,
   copyPageButton: z.boolean().default(true).meta({
     description: "Hide/show copy page content button."
   }),
